instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/Servlet3OutputStreamWriteBytesAndOffsetAdvice.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0;
+
+import static io.opentelemetry.javaagent.instrumentation.servlet.v3_0.Servlet3Singletons.getSnippetInjectionHelper;
+import static io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet.Injection.getInjectionState;
+
+import io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet.InjectionState;
+import java.io.IOException;
+import javax.servlet.ServletOutputStream;
+import net.bytebuddy.asm.Advice;
+
+public class Servlet3OutputStreamWriteBytesAndOffsetAdvice {
+  @Advice.OnMethodEnter(skipOn = Advice.OnDefaultValue.class, suppress = Throwable.class)
+  public static boolean methodEnter(
+      @Advice.This ServletOutputStream servletOutputStream,
+      @Advice.Argument(value = 0) byte[] write,
+      @Advice.Argument(value = 1) int off,
+      @Advice.Argument(value = 2) int len)
+      throws IOException {
+    InjectionState state = getInjectionState(servletOutputStream);
```

Comment:
(same for other calls of getInjectionState)
```suggestion
    InjectionState state = Injection.getInjectionState(servletOutputStream);
```